### DISCUSSION TITLE
using-octokit-graphql-to-interact-with-the-github-discussions-api/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#434] [Announcements]
# using-octokit-graphql-to-interact-with-the-github-discussions-api/

In this post I describe a .NET script/tool I created to migrate the comments on my blog posts from my legacy Disqus account to GitHub discussions

https://andrewlock.net/using-octokit-graphql-to-interact-with-the-github-discussions-api/

**Comment by 74hz** [comment]
Please help me with the 404 problem in giscus
